### PR TITLE
Sync Serverless CI config/

### PR DESCRIPTION
**Pull request by serverless-qe**

### CODE REVIEW

File: ci-operator/config/openshift-knative/eventing-istio/openshift-knative-eventing-istio-release-v1.15__415.yaml [hunk 1]
```diff
@@ -1,8 +1,24 @@
...
+    tag: rhel-8-release-golang-1.22-openshift-4.17
 build_root:
   project_image:
     dockerfile_path: openshift/ci-operator/build-image/Dockerfile
 images:
 - dockerfile_path: openshift/ci-operator/knative-images/controller/Dockerfile
+  inputs:
+    ocp_ubi-minimal_8:
+      as:
+      - $GO_RUNTIME
```
Comment (skonto) [inline ci-operator/config/openshift-knative/eventing-istio/openshift-knative-eventing-istio-release-v1.15__415.yaml:18]: Do we set those somewhere else?

Comment (dsimansk) [reply ci-operator/config/openshift-knative/eventing-istio/openshift-knative-eventing-istio-release-v1.15__415.yaml:18]: Through hack's image template: https://github.com/openshift-knative/hack/blob/main/cmd/generate/dockerfile-templates/DefaultDockerfile.template

Comment (pierDipi) [reply ci-operator/config/openshift-knative/eventing-istio/openshift-knative-eventing-istio-release-v1.15__415.yaml:18]: Yes, now every component defines those or is about to start defining them

**Pull request merged**